weaver.toml

```diff
@@ -11,11 +11,26 @@ binary = "template"
 # env = ["PUT=foo", "ENV=bar", "VARS=baz", "HERE="]
 
 # The [single] section configures the behavior of the app when deployed via
-# `weaver single deploy`.
+# `weaver single deploy`. See serviceweaver.dev/docs.html#single-process for
+# more information about the single process deployer.
 [single]
 listeners.lis = {address="localhost:9000"}
 
 # The [multi] section configures the behavior of the app when deployed via
-# `weaver multi deploy`.
+# `weaver multi deploy`. See serviceweaver.dev/docs.html#multiprocess for more
+# information about the multiprocess deployer.
 [multi]
 listeners.lis = {address="localhost:9000"}
+
+# The [kube] section configures the behavior of the app when deployed via
+# `weaver kube deploy`. See github.com/serviceweaver/weaver-kube for
```

Comment:
can we add a TODO to point out to serviceweaver.dev/docs.html#kube instead once we add the kube documentation in the docs?

Reply:
I don't want to add the TODO here since the user will see it when they run `gonew`, but I'll keep a mental TODO.